cypress/integration/page.spec.js

```diff
@@ -721,7 +760,7 @@
       });
 
       it('should not show NewCommentForm', () => {
-        newCommentForm.el().should('not.exist')
+        newCommentForm.el().should('not.exist');
```

Comment:
The `newCommentForm.el().should('not.exist')` assertion is correct, but ensure that the form is indeed not supposed to exist at this point in the test.